subsys/caf/modules/sensor_sampler.c

```diff
@@ -313,6 +321,8 @@ static int sensor_init(void)
 		sd->sampling_period = sc->sampling_period_ms;
 		sd->sample_timeout = cur_uptime + sc->sampling_period_ms;
 
+		sd->active_sensor_events_limit = sc->active_events_limit;
```

Comment:
I wonder if we need to keep copy of this value. Maybe we could use the value from sensor configuration directly?

Reply:
Or we expect to make if configurable at runtime at some point?